README.md

```diff
@@ -12,7 +12,7 @@ The following table shows the relation between the versions of the two projects:
 
 | NGINX Ingress Controller | NGINX Ingress Operator |
 | --- | --- |
-| 2.1.x | 0.5.0 |
+| 2.1.x | 0.5.1 |
```

Comment:
Since we're releasing a minor version, is it more like below?
2.1.1 0.5.1
2.1.0 0.5.0

Reply:
I wasn't really sure how to do it since we don't usually call out minor versions of KIC